docs/resources/azure_policy_definition.md

```diff
@@ -0,0 +1,107 @@
+---
+title: About the azure_policy_definition Resource
+platform: azure
+---
+
+# azure_policy_definition
+
+Use the `azure_policy_definition` InSpec audit resource to test properties and configuration of an Azure policy definition.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used .
+For more information, refer to the resource pack [README](../../README.md). 
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure). 
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+`name` or the `resource_id` must be given as a parameter.
+```ruby
+describe azure_policy_definition(name: 'my_policy') do
+  it { should exist }
+end
+```
+```ruby
+describe azure_policy_definition(resource_id: '/subscriptions/{subscriptionId}/providers/Microsoft.Authorization/policyDefinitions/{policyDefinitionName}') do
+  it { should exist }
+end
+```
+## Parameters
+
+| Name                                  | Description |
+|---------------------------------------|-------------|
+| name                                  | Name of the policy definition. `policyDefinitionName` |
+| built_in                              | Indicates whether the policy definition is built-in. Optional, default is `false`. This should not be used when `resource_id` is provided. |
```

Comment:
Is this saying that this field is optional, and it will default to `false`? If so, this might read slightly better.
```
Indicates whether the policy definition is built-in. Optional. Defaults to `false` if not supplied. This should not be used when `resource_id` is provided.
```